.travis.yml

```diff
@@ -72,10 +72,14 @@ matrix:
           packages:
             - nginx
 
+before_cache:
+  - mkdir $HOME/dojo_archive 2>/dev/null
```

Comment:
rather than redirecting std err to /dev/null I'd prefer to see 
```bash
mkdir -p $HOME/dojo_archive
```
That way the mkdir is silent if the dir already exists or is successfully created, but if the creation fails we will see any errors